zilliqa/src/pool.rs

```diff
@@ -176,7 +263,30 @@ mod tests {
                 },
             },
             signer: from_addr,
-            hash: Hash::ZERO,
+            hash: Hash::compute([from_addr.as_bytes(), &[nonce]]),
```

Comment:
Is this the "official" txn hash? If it is, shouldn't it be keccak256 of all txn fields of the signed transaction?